src/tests/ftest/dfuse/fio_pil4dfs_small.py

```diff
@@ -0,0 +1,53 @@
+"""
+  (C) Copyright 2019-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from fio_test_base import FioBase
+from fio_utils import FioCommand
+
+class FioILSmall(FioBase):
+    """Test class Description: Runs Fio with in small config.
+
+    :avocado: recursive
+    """
+
+    def setUp(self, il_lib_path=None):
+        """Set up each test case."""
+        # obtain separate logs
+        self.update_log_file_names()
+
+        # Start the servers and agents
+        super().setUp()
+
+        # Get the parameters for Fio
+        self.fio_cmd = FioCommand(il_lib_path='/usr/lib64/libpil4dfs.so')
+        self.fio_cmd.get_params(self)
```

Comment:
Better to not add a param for a specific env variable
```suggestion
        self.fio_cmd = FioCommand()
        self.fio_cmd.get_params(self)
        self.fio_cmd.env['LD_PRELOAD'] = '/usr/lib64/libpil4dfs.so'
```

Reply:
Thank you very much! Fixed as you suggested.